lib/iiif_manifest/v3/manifest_builder/canvas_builder.rb

```diff
@@ -71,6 +82,12 @@ def attach_content
             choice_builder.new(display_content).apply(canvas)
           end
         end
+
+        def iiif_endpoint
+          display_image.try(:iiif_endpoint) ||
```

Comment:
`.try(:iiif_endpoint)` and `&.iiif_endpoint` are semantically the same.  Favor `&.` instead of `.try`

History lesson: `.try` precluded the lonely operator (e.g. `&.`) as a Rails idiom.  Then overtime Ruby (and perhaps Rails) added the `&.` because `.try` had become ubiquitous and there could be a more "elegant" approach.